### PR TITLE
0.7.1

### PR DESCRIPTION
**Pull request by oblm**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
This PR merges the tested `0.7.1-rc` into `preview`.

#### Release note
See CHANGELOG.md

#### Tests
Local deploy, cloud deploy and tested the functionality of the CN region workflows (as far as they can be tested with online services not yet fully working).

#### Documentation
See CHANGELOG.md
There is also a follow up docs ticket here: [DOC-1471
](https://improbableio.atlassian.net/browse/DOC-1471)
#### Primary reviewers
@improbable-danny

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,16 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [Unreleased-`x.y.z`] - 2019-xx-xx
 
+## [`0.7.1-preview`] - 2019-12-06
+
+### Bug fixes: 
+- The C Worker SDK now communicates on port 443 instead of 444. This change is intended to protect your cloud deployments from DDoS attacks.
+
+### Internal:
+Features listed in the internal section are not ready to use but, in the spirit of open development, we detail every change we make to the GDK.
+- The GDK is now compatible with the `CN` launch region. When Improbable's online services are fully working in China, they will work with this version of the GDK. You will be able to create SpatialOS Deployments in China by specifying the `CN` region in the Deployment Launcher.
```
Comment (ernopp) [inline CHANGELOG.md]: When Improbable's platform is fully working in China

Comment (ernopp) [reply CHANGELOG.md]: it's about more than just OS (in fact we won't have the OS for a while, just the deployment platform)

Comment (oblm) [reply CHANGELOG.md]: resolved offline

File: SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp [hunk 2]
```diff
@@ -14,7 +14,21 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialDeploymentManager);
 
-static const FString SpatialServiceVersion(TEXT("20190930.180414.3b04a59226"));
+static const FString SpatialExe(TEXT("spatial.exe"));
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp]:
```suggestion
```

**Pull request merged**